src/dispatch/static/dispatch/src/components/HoverCard.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
This looks specific to signals, should it be named `SignalHoverCard` and/or moved under `/signals`

Reply:
I removed it for now.